packages/server/src/arpa_reporter/db/reporting-periods.js

```diff
@@ -84,8 +74,8 @@ async function getReportingPeriodID(periodID) {
  * @returns The matching reporting periods, sorted from oldest to newest by date
  */
 async function getPreviousReportingPeriods(period_id, trns = knex) {
-    const currentReportingPeriod = await getReportingPeriod(period_id);
-    const allReportingPeriods = await getAllReportingPeriods();
+    const currentReportingPeriod = await getReportingPeriod(period_id, trns);
+    const allReportingPeriods = await getAllReportingPeriods(trns);
```

Comment:
This is a functional change, but as far as I can tell no callsite of `getPreviousReportingPeriods` was passing a transaction, so I don't think it has a real world effect.